File: CodeWars/Card_Game/sol.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
   int f, p;
   cin >> f >>  p;
   if (f == p) {
      cout << "Draw";
      return 0;
   }
   if (f == 1) {
      cout << "Ferb";
   } else if (p == 1) {
      cout << "Phineas";
   } else {
      if (f < p) {
         cout << "Phineas";
      } else {
         cout << "Ferb";
      }
   }
}

File: CodeWars/Card_Game/sol.py
f, p = map(int, input().split())
if f == p:
   print("Draw")
   exit(0)
if f == 1:
   print("Ferb")
elif p == 1:
   print("Phineas")
else:
   if p < f:
      print("Ferb")
   else:
      print("Phineas")